src/bowling.js

```diff
@@ -0,0 +1,89 @@
+import Frame from './frame.js';
```

Comment:
Nice conversion of your ruby code to javascript. Really readable and easy to understand whats going on.